package.json

```diff
@@ -35,11 +35,12 @@
   },
   "devDependencies": {
     "@babel/core": "^7.12.10",
-    "@storybook/addon-essentials": "6.1.17",
-    "@storybook/addon-links": "6.1.17",
-    "@storybook/addons": "6.1.17",
-    "@storybook/react": "6.1.17",
-    "@storybook/theming": "6.1.17",
+    "@storybook/addon-a11y": "^6.1.18",
```

Comment:
Why the `^` only on this one ?

Reply:
I will setup a GitHub Action for that, it does not matter!